.travis/smoke-test.sh

```diff
@@ -24,6 +24,9 @@ echo "Smoke testing api-gateway (scholarly-articles content-store)"
 [[ "$(curl -sS "http://localhost:${HTTP_PORT_GATEWAY}/scholarly-articles/ping" 2>&1)" == "pong" ]]
 [[ "$(curl -sS "http://localhost:${HTTP_PORT_GATEWAY}/scholarly-articles/items" --output /dev/null --write-out '%{http_code}' 2>&1)" == "200" ]]
 
+echo "Smoke testing api-gateway (search)"
+[[ "$(curl -sS "http://localhost:${HTTP_PORT_GATEWAY}/search/ping" 2>&1)" == "pong" ]]
```

Comment:
I have used [`uwsgi_curl`](https://github.com/andreif/uwsgi-tools) before but it needs to be installed in the container image or virtualenv, opinions @GiancarloFusiello?

Reply:
What would a uwsgi healthcheck consist of?

Reply:
It's like calling `/ping` but through the WSGI protocol directly, rather than going through HTTP and nginx first.
Here's an [extensive example from `profiles`](https://github.com/elifesciences/profiles/blob/develop/smoke_tests_wsgi.sh) that does much more than a single call, with the same library.

Reply:
what's the benefit of skipping nginx?

Reply:
I think the value is knowing if nginx configuration is faulty vs. the service itself is down

Reply:
Yep (this healthcheck would be in the image). 

Can then be added to https://github.com/libero/sample-configuration/blob/cd9c645681a5c1da0836ca2322b9c0c73068e28a/.travis/smoke-test.sh#L8-L17

Reply:
Ok, but by definition won't be in this PR as the image needs to be modified.

Reply:
https://github.com/libero/search/pull/13 has a basic healthcheck on the port being listened on, so that we can add the container in this list.